flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/utils/TemporalJoinUtil.scala

```diff
@@ -198,7 +198,7 @@ object TemporalJoinUtil {
   }
 
   def isRowTimeTemporalJoinConditionCall(rexCall: RexCall): Boolean = {
-    rexCall.getOperator == TEMPORAL_JOIN_CONDITION && rexCall.operands.length > 3
+    rexCall.getOperator == TEMPORAL_JOIN_CONDITION && rexCall.operands.length == 5
```

Comment:
Why the 4 operands call `(LEFT_TIME_ATTRIBUTE, RIGHT_TIME_ATTRIBUTE, LEFT_KEY, RIGHT_KEY)` is not row time temporal join?

Reply:
the PRIMARY_KEY is necessary for row time temporal join.